spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+feature 'user deletes their own review', %Q{
+  As a signed in user
+  I want to delete my own reviews
+  So that I can erase my embarrassing reviews
+} do
+  scenario 'edit trails review' do
+    testuser = FactoryBot.create(:user, username: "testuser")
+    trail = FactoryBot.create(:trail)
+
+    visit new_user_session_path
+
+    fill_in 'Email', with: testuser.email
+    fill_in 'Password', with: testuser.password
+    click_button 'Log in'
+
+    visit new_trail_review_path(trail.id)
+    fill_in 'Rating', with: 1
+    fill_in 'Comment', with: 'Great trail'
+    click_button 'Add Review'
```

Comment:
I am okay with merging as-is, but you should be able to create the new review via a factory and simply go through the steps of signing in and clicking the delete button in your test itself